modules/ROOT/pages/administration/access-control/database-administration.adoc

```diff
@@ -182,6 +182,10 @@ REVOKE [IMMUTABLE] database-privilege ON { HOME DATABASE \| DATABASE[S] { * \| n
 Use `REVOKE` if you want to remove a privilege.
 ====
 
+Common errors such as misspellings or attempting to grant privileges to roles where either does not exist will lead to notifications.
```

Comment:
granting to non-existing will fail not give notifications, I guess it should say revoke?